tests/sentry/models/test_activity.py

```diff
@@ -15,6 +17,70 @@ def test_get_activities_for_group_none(self):
         assert len(act_for_group) == 1
         assert act_for_group[0].type == ActivityType.FIRST_SEEN.value
 
+    @with_feature("projects:issue-priority")
+    def test_get_activities_for_group_simple_priority_ff_on(self):
+        project = self.create_project(name="test_activities_group")
+        group = self.create_group(project)
+        user1 = self.create_user()
+
+        activities = [
+            Activity.objects.create_group_activity(
+                group=group,
+                type=ActivityType.SET_UNRESOLVED,
+                user=user1,
+                data=None,
+                send_notification=False,
+            ),
+            Activity.objects.create_group_activity(
+                group=group,
+                type=ActivityType.SET_PRIORITY,
+                user=user1,
+                data={"priority": PRIORITY_LEVEL_TO_STR[PriorityLevel.LOW]},
+                send_notification=False,
+            ),
+            Activity.objects.create_group_activity(
+                group=group,
+                type=ActivityType.SET_PRIORITY,
+                user=user1,
+                data=None,
+                send_notification=False,
+            ),
+        ]
+
+        act_for_group = Activity.objects.get_activities_for_group(group=group, num=100)
+        assert len(act_for_group) == 3
+        assert act_for_group[0] == activities[-2]
+        assert act_for_group[1] == activities[-3]
+        assert act_for_group[-1].type == ActivityType.FIRST_SEEN.value
```

Comment:
We also want to assert that the activity entry has a `priority` value in the FIRST_SEEN entry ([this part of the tech spec](https://www.notion.so/sentry/Tech-Spec-Priority-in-Issue-Details-Stream-ac9ecbbc5cfd4baaae39ecf32f8e74ce?pvs=4#df73cb01452443608b7883d3929c8e4d))

Reply:
ooof lol forgot